app/routers/api/notifiers.py

```
@@ -71,10 +72,32 @@ def create_notifier(notifier: NotifierCreate, db: Session = Depends(get_db)):
+        subscribed_events=notifier.subscribed_events,
+        smtp_host=notifier.smtp_host,
+        smtp_port=notifier.smtp_port,
+        smtp_user=notifier.smtp_user,
+        smtp_password=notifier.smtp_password,  # Property setter handles encryption
+        smtp_sender=notifier.smtp_sender,
+        smtp_use_tls=notifier.smtp_use_tls,
+    )
     db.add(db_notifier)
     db.commit()
     db.refresh(db_notifier)
+    logger.info(f"Created notifier '{notifier.name}' ({notifier.type})")
```

Ruff (0.14.13)

Logging statement uses f-string

(G004)